### ISSUE TITLE
Unicode Support

### ISSUE THREAD

**Issue by dmtucker** [sarenji/pyrc#24]
Please :)

**Comment by wei2912** [comment]
Python 3 supports Unicode a lot better and there's already a patch that
converts pyrc to py3; perhaps you could give it a try? :)

**Comment by wei2912** [comment]
Hmm, seems like @sarenji isn't active. Would anyone mind if I were to maintain a Python 3 fork of pyrc?

**Comment by sarenji** [comment]
@wei2912 I'm happy to make you maintainer of this repo, since I don't have time to. Interested?

**Comment by wei2912** [comment]
@sarenji Sure, I'd willingly take it up. :) I would probably convert it to Python 3 though; would you be fine with it?

**Comment by sarenji** [comment]
@wei2912 Yep, that's fine. Ideally there'd be support for python2, but in the absence of it, python3 is the way forward.

Can you also message me your PyPI username?

**Comment by sarenji** [comment]
Also, when you convert it to Python3, that probably warrants a major version upgrade according to semver :)

**Comment by wei2912** [comment]
Same username as this one.

**Comment by sarenji** [comment]
@wei2912 actually, could you add your maintainer info to `setup.py`? then i can update the package!

**Comment by wei2912** [comment]
@sarenji Sure, on it. :)

**Comment by wei2912** [comment]
@sarenji Done: https://github.com/sarenji/pyrc/commit/42bd22eace4b921d2dfbc6e3da9b0d83793644c4 :) Thanks for the opportunity!

**Comment by sarenji** [comment]
@wei2912 :+1: :smile:
